### PR TITLE
[5.2] No menu must be selected for menu items batch processing

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #43215 #43214 

### Summary of Changes
Batch processing of menu items seems to be possible for all menu items, also if no menu is selected.
This PR removes the condition, that a menu must be selcted. 

### Testing Instructions
The more menus you have and the more options like multilingual, many leveles in the menus, the better.
Mark some menuItems - if you want select different levels, different languages, 
Activate the batch processing in the actions menu.
Change language, access level, 
copy or move the selecten menutems.

Make sure that 
- wrong moves give error messages
- all other moves are performed correctly

### Actual result BEFORE applying this Pull Request
If no menu is selected, a message is displayed
COM_MENUS_SELECT_MENU_FIRST="To use batch processing, please first select a Menu in the manager."

### Expected result AFTER applying this Pull Request
The batch processing is possible, also if no menu is selcted. 

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 9747667a0a0e2761406d384e4d120e20fb031700<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44286">issues.joomla.org/tracker/joomla-cms/44286</a>.</sub>

**Comment by fgsw** [comment]
I have tested this item :white_check_mark: successfully on 9747667a0a0e2761406d384e4d120e20fb031700<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44286">issues.joomla.org/tracker/joomla-cms/44286</a>.</sub>

File: administrator/components/com_menus/tmpl/items/default_batch_body.php [hunk 1]
```diff
@@ -34,23 +32,22 @@
...
+    <?php if ($clientId != 1) : ?>
+        <div class="row">
+                <?php if (Multilanguage::isEnabled()) : ?>
+                <div class="form-group col-md-6">
+                    <div class="controls">
+                        <?php echo LayoutHelper::render('joomla.html.batch.language', []); ?>
+                    </div>
+                </div>
+                <?php endif; ?>
             <div class="form-group col-md-6">
```
Comment (Quy) [inline administrator/components/com_menus/tmpl/items/default_batch_body.php]:
```suggestion
            <?php if (Multilanguage::isEnabled()) : ?>
                <div class="form-group col-md-6">
                    <div class="controls">
                        <?php echo LayoutHelper::render('joomla.html.batch.language', []); ?>
                    </div>
                </div>
            <?php endif; ?>
            <div class="form-group col-md-6">
```

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44286">issues.joomla.org/tracker/joomla-cms/44286</a>.</sub>

**Comment by pe7er** [comment]
Thanks @chmst !

**Pull request merged**